rules/nette-code-quality/src/NodeAdding/FunctionLikeFirstLevelStatementResolver.php

```diff
@@ -57,9 +57,8 @@ public function resolveFirstLevelStatement(Node $node): Node
      */
     private function matchMultiplierClosure(Node $node): ?Closure
     {
-        /** @var Closure|null $closure */
         $closure = $node->getAttribute(AttributeKey::CLOSURE_NODE);
-        if ($closure === null) {
+        if (! $closure instanceof \Closure) {
```

Comment:
this is false positive as there is already use PhpParser\Node\Expr\Closure; means \Closure is invalid.

Reply:
fixed.